### ISSUE TITLE
pickle.UnpicklingError: invalid load key,

### ISSUE THREAD

**Issue by inaryart** [CorentinJ/Real-Time-Voice-Cloning#962]
- latest version
```
  python demo_cli.py
Arguments:
    enc_model_fpath:   saved_models\default\encoder.pt
    syn_model_fpath:   saved_models\default\synthesizer.pt
    voc_model_fpath:   saved_models\default\vocoder.pt
    cpu:               False
    no_sound:          False
    seed:              None

Running a test of your configuration...

Found 1 GPUs available. Using GPU 0 (NVIDIA GeForce GTX 1650 with Max-Q Design) of compute capability 7.5 with 4.3Gb total memory.

Preparing the encoder, the synthesizer and the vocoder...
                                                                                                                       Loaded encoder "encoder.pt" trained to step 1564501
Synthesizer using device: cuda
Building Wave-RNN
Trainable Parameters: 4.481M
Loading model weights at saved_models\default\vocoder.pt
Testing your configuration with small inputs.
        Testing the encoder...
        Testing the synthesizer... (loading the model will output a lot of text)
Trainable Parameters: 30.870M
Traceback (most recent call last):
  File "D:\test\Real-Time-Voice-Cloning\demo_cli.py", line 95, in <module>
    mels = synthesizer.synthesize_spectrograms(texts, embeds)
  File "D:\test\Real-Time-Voice-Cloning\synthesizer\inference.py", line 86, in synthesize_spectrograms
    self.load()
  File "D:\test\Real-Time-Voice-Cloning\synthesizer\inference.py", line 64, in load
    self._model.load(self.model_fpath)
  File "D:\test\Real-Time-Voice-Cloning\synthesizer\models\tacotron.py", line 496, in load
    checkpoint = torch.load(str(path), map_location=device)
  File "C:\Users\NAMW\AppData\Local\Programs\Python\Python39\lib\site-packages\torch\serialization.py", line 608, in load
    return _legacy_load(opened_file, map_location, pickle_module, **pickle_load_args)
  File "C:\Users\NAMW\AppData\Local\Programs\Python\Python39\lib\site-packages\torch\serialization.py", line 777, in _legacy_load
    magic_number = pickle_module.load(f, **pickle_load_args)
_pickle.UnpicklingError: invalid load key, ' '.
```

**Comment by sourceful-wing** [comment]
Getting same issue

**Comment by CorentinJ** [comment]
I see, the synthesizer download is failing. You can download it manually [here](https://drive.google.com/u/0/uc?export=download&id=1EqFMIbvxffxtjiVrtykroF6_mUh-5Z3s) while I find a fix.

**Comment by CorentinJ** [comment]
Fixed in #963
